osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/application/config/AbstractSchedulingConfig.java

```diff
@@ -117,12 +176,18 @@ private Properties constructQuartzConfiguration(final SchedulingConfigProperties
         return properties;
     }
 
+    @Deprecated
     private JobDetail createJobDetail(final Class<? extends Job> jobClass) {
         return JobBuilder.newJob().ofType(jobClass).storeDurably().withIdentity(jobClass.getSimpleName()).build();
     }
 
+    @Deprecated
     private Trigger createJobTrigger(final JobDetail jobDetail, final String cronExpression) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1133)